skopt/optimizer/__init__.py

```diff
@@ -0,0 +1,4 @@
+from . dummy_opt import dummy_minimize
+from . gbrt_opt import gbrt_minimize
+from . forest_opt import forest_minimize
+from . gp_opt import gp_minimize
```

Comment:
What is the difference to `from .gpopt import gp_min` which we use in other places? #stylepolice